decoderTest/decoderTest.csproj

```diff
@@ -13,4 +13,8 @@
     <NoWarn>1701;1702;3026</NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="..\defaults\defaults.clog.cs" Link="defaults.clog.cs" />

```

Comment:
Why does decoderTest need a change?

Reply:
Because I don't want 2 copies of that file. This doesn't change the behavior of that project, it just uses the defaults file from the new directory.